develop/dev-guide-sample-application-aws-lambda.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Connect to TiDB by using mysql2 in AWS Lambda Function
+summary: This article describes how to connect TiDB using mysql2 in AWS Lambda Function and provides a simple example code snippet.
+---
+
+<!-- markdownlint-disable MD024 -->
+<!-- markdownlint-disable MD029 -->
+
+# Connect to TiDB by using mysql2 in AWS Lambda Function
+
+TiDB is a MySQL-compatible database.
+
+[mysql2](https://www.npmjs.com/package/mysql2) is one of the most popular open-source Node.js drivers.
+
+[AWS Lambda Function](https://aws.amazon.com/lambda/) is a serverless computing service provided by AWS.
+
+This document shows you how to use TiDB and mysql2 to build a simple CRUD application.
+
+## Prerequisites
+
+- [Node.js **18**](https://nodejs.org/en/download/) or later.
+- [Git](https://git-scm.com/downloads).
+- TiDB cluster. If you don't have a TiDB cluster, you can create one by following the steps below:
+    - (Recommended) Refer to [Create a TiDB Serverless cluster](/develop/dev-guide-build-cluster-in-cloud.md#step-1-create-a-tidb-serverless-cluster) to create your own TiDB Cloud cluster.
+    - Refer to [Deploy a local test TiDB cluster](/quick-start-with-tidb.md#deploy-a-local-test-cluster) or [Deploy a formal TiDB cluster](/production-deployment-using-tiup.md) to create a local cluster.
+- AWS account. If you don't have an AWS account, you can create one by following the steps below:
+    - Refer to [Create an AWS Lambda Function](https://docs.aws.amazon.com/lambda/latest/dg/getting-started.html) to create an AWS Lambda Function.
+
```

Comment:
To complete this guide, you need:

- [Node.js](https://nodejs.org/en/download/)  18 or later
- [Git](https://git-scm.com/downloads)
- A TiDB cluster running
- [An AWS account](https://portal.aws.amazon.com/billing/signup)
- An AWS user can access the Lambda function

**If you don't have a TiDB cluster yet, please create one with one of the following methods:**

- (Recommended) Refer to [Create a TiDB Serverless cluster](/develop/dev-guide-build-cluster-in-cloud.md#step-1-create-a-tidb-serverless-cluster) to create your own TiDB Cloud cluster.
- Refer to [Deploy a local test TiDB cluster](/quick-start-with-tidb.md#deploy-a-local-test-cluster) or [Deploy a formal TiDB cluster](/production-deployment-using-tiup.md) to create a local cluster.

**If you don't have an AWS account and user, you can create them by following the steps  in the [Getting started with Lambda](https://docs.aws.amazon.com/lambda/latest/dg/getting-started.html) guide.**